org.musicbrainz.Picard.json

```diff
@@ -180,11 +248,31 @@
                 }
             ]
         },
+        {
+            "name": "chromaprint",
+            "cmake": true,
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE:STRING=Release",
+                "-DBUILD_TOOLS:BOOL=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/acoustid/chromaprint/releases/download/v1.4.3/chromaprint-1.4.3.tar.gz",
+                    "sha256": "ea18608b76fb88e0203b7d3e1833fb125ce9bb61efe22c6e169a50c52c457f82"
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig"
+            ]
+        },
```

Comment:
Does Chromaprint really need to be moved? (as usual, if it does, please move it in its own commit, otherwise please keep it where it is)